package.json

```diff
@@ -48,6 +48,7 @@
     "canvas": "^2.6.1",
     "chart.js": "^2.9.2",
     "cheerio": "^1.0.0-rc.3",
+    "classnames": "^2.2.6",
```

Comment:
Since we're introducing `classnames`, at some point we probably want to go through the codebase and introduce it wherever it makes sense. That's a separate issue for another time though.